src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss

```diff
@@ -18,12 +18,12 @@ dot-whats-changed,
 }
 
 .dot-edit__page-wrapper {
-    align-self: center;
-    display: flex;
-    flex-direction: column;
-    flex-grow: 1;
-    justify-content: center;
-    padding: 0 $basic-padding * 2 $basic-padding * 2;
+    height: 100%;
+    max-width: 100%;
+    overflow: scroll;
+    min-width: 100%;
+    width: 100%;
+    padding: $basic-padding * 4;
```

Comment:
I think this padding should be removed, since it's adding extra space
![image](https://user-images.githubusercontent.com/37185433/92049673-489e8d00-ed48-11ea-9cae-860703c586cb.png)

Reply:
Ouuuuuu! is not remove it, but is added only we are deviced.